package/README.md

```diff
@@ -57,6 +57,8 @@ Here is an overview of the release process. See the section below for detailed i
 
    * When done, put release notes into a private Gist and send the link to other maintainers for review.
 
+   * Update versions in cores/esp8266/core_esp8266_version.h -and- platform.txt, and commit
```

Comment:
Perhaps a CI check is needed for this. I.e. if commit is tagged, and tag doesn't match the contents of core_esp8266_version.h or platform.txt, then CI fails.

Reply:
Not anymore needed, doc will be upated